cluster-state-service/handler/api/v1/taskapis.go

```diff
@@ -24,19 +24,22 @@ import (
 	"github.com/blox/blox/cluster-state-service/handler/store"
 	"github.com/blox/blox/cluster-state-service/handler/types"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 )
 
 const (
 	taskARNKey     = "arn"
 	taskClusterKey = "cluster"
 
-	taskStatusFilter  = "status"
-	taskClusterFilter = "cluster"
+	taskStatusFilter    = "status"
+	taskClusterFilter   = "cluster"
+	taskStartedByFilter = "startedBy"
```

Comment:
Sadly, [Docker](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/) isn't consistent in the query param convention (eg: `ps_args` in `/containers/{id}/top` vs `detachKeys` in `/containers/{id}/start`.  [Kubernetes](http://kubernetes.io/docs/api-reference/v1/operations/) consistently uses CamelCase for query parameters and mashedwords for path segments (eg `/api/v1/componentstatuses`).  So, there's no conventional guidance from prominent golang operational tooling.